exercises/03-xpath-intro.md

```diff
@@ -9,47 +9,50 @@
 
 Ideally, web development and test automation goes hand in hand, so
 that the people who develop the websites, would also do (at least
-a bit of) test automation. However, more often than not this is not
-the case and that causes trouble to test automation developers.
-Websites are usually creates "UX first", which is good for the users,
+a bit of) test automation, or be aware of it. However, more often than not this is not
+the case and that causes trouble for test automation developers.
+Websites are often created with "UX first" approach, which is good for the users,
 but it typically means more work for the test automation developers.
 
-Nowadays, websites are created with modern frameworks, which generates
-nice looking websites from templates. Most often this causes developers
-to forget to add unique `id` attributes to elements, so we need
+Nowadays, websites are created with modern frameworks, which generate
+nice looking websites from templates. Pretty often developers
+forget to add unique `id` attributes to elements, so we need
 alternative locators for our elements.
 
-This is where XPaths step in. Sure enough, there are other locators
-with SeleniumLibrary and Browser than XPaths and IDs, but quite often using an XPath is the
-only way to find a truly unique identifier for an element.
+This is where XPaths step in. Sure enough, there are other locator strategies
```

Comment:
I would very much drive here the point home that you should pretty much always use some other locator strategies than XPath. It is only needed when the only identifiable information of an element is the visible text in the element. 

(With Browser, this gets even easier with composable selectors, so you virtually never need XPath)

Identifying element in any other way usually means you avoid XPath. I agree that the training should have this section, but I would just emphasise here that XPath should be last resort.